lemonparty2018.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from flask import render_template
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+if __name__ == '__main__':
+    app.run(debug=True, host='0.0.0.0')
```

Comment:
We should make `debug` configurable (in a local settings file, or an environment variable)